.circleci/config.yml

```diff
@@ -87,13 +87,37 @@ jobs:
           root: ~/mibitracker-client
           paths:
             - docs
+  tag_test:
+    working_directory: ~/mibitracker-client
+    docker:
+      - image: circleci/python:3.7 # container for the build job
 
+    steps:
+      - checkout:
+          path: ~/mibitracker-client
+      - run:
+          name: Check if tag and setup.py are consistent
+          command: |
+            python check_tag.py
```

Comment:
I assume you confirmed that this works without any python environment or additional libraries, correct?

Reply:
Yes. It works on github.